server/core/routing/graph_builder.py

```
@@ -221,3 +224,75 @@ def densify_route_path(G: nx.MultiDiGraph, node_path: list[int]) -> list[dict]:
             points.append({'lat': G.nodes[v]['y'], 'lng': G.nodes[v]['x']})
     return points
 
+def add_random_traffic_to_subgraph(subgraph: nx.MultiDiGraph, min_factor=0.7, max_factor=1.5):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)